executor_test.go

```diff
@@ -675,6 +675,48 @@ func TestExecutor_Execute_Clear(t *testing.T) {
 			t.Fatalf("expected column changed")
 		}
 	})
+
+	t.Run("RowKeyColumnKey_NotClearNot", func(t *testing.T) {
+		writeQuery := `Set("one", f="ten")`
+		readQueries := []string{
+			`Not(Row(f="whatever"))`,
+			`Clear("one", f="whatever")`,
```

Comment:
would it make more sense to clear "ten" and show that even after that Not(whatever) returns true? ...or maybe we add that case after this one so there'd be two more lines, one for `Clear("one", f="ten")` and one doing Not(whatever) again.

Reply:
@jaffee - done, PTAL